cypress/objects/taskPage.js

```diff
@@ -0,0 +1,36 @@
+class Tasks {
+    getTasks() {
+        return cy.get('[href="/tasks"]');
+    }
+    getButton() {
+        return cy.get('.btn');
+    }
+    getFormName() {
+        return cy.get('[id="taskDetailsForm.name"]');
+    }
+    getFieldTextArea() {
+        return cy.get('.field__textarea');
+    }
+    getValue() {
+        return cy.get('.Select-value-label');
```

Comment:
need more accurate name than generic getValue, I need to understand what I get here